### PR TITLE
Add FCnudged_GC component set based on FCnudged.

### PR DESCRIPTION
**Pull request by jimmielin**
Only change is in `config_compsets.xml` which adds `FCnudged_GC` based on `FCnudged` definition (with `%NUDG`)

### CODE REVIEW

File: bld/namelist_files/use_cases/hist_geoschem_nudged.xml [hunk 1]
```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0"?>
+
+<namelist_defaults>
+
+<!-- Settings in this file were adapted from hist_trop_strat_vbs_cam6.xml -->
```
Comment (lizziel) [inline bld/namelist_files/use_cases/hist_geoschem_nudged.xml]: Should this be hist_trop_strat_nudged_cam6.xml?

File: bld/namelist_files/use_cases/hist_geoschem_nudged.xml [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+<use_gw_convect_dp>.true.</use_gw_convect_dp>
+<tau_0_ubc>.false.</tau_0_ubc>
+<gw_qbo_hdepth_scaling>0.25D0</gw_qbo_hdepth_scaling>
+
+<flbc_type>SERIAL</flbc_type>
+<flbc_file>atm/waccm/lb/LBC_1750-2014_CMIP6_0p5degLat_c170126.nc</flbc_file>
+
+<prescribed_strataero_type>SERIAL</prescribed_strataero_type>
+
+<ndep_list>'noy', 'nhx'</ndep_list>
```
Comment (lizziel) [inline bld/namelist_files/use_cases/hist_geoschem_nudged.xml:35]: The nudged file also contains these:
<ext_frc_type>INTERP_MISSING_MONTHS</ext_frc_type>
<srf_emis_type>INTERP_MISSING_MONTHS</srf_emis_type>
I know we don't need these for GEOS-Chem but could you remind me why?

**Review by lizziel — COMMENTED** [review]
Looks good!

File: bld/namelist_files/use_cases/hist_geoschem_nudged.xml (hunk 2 above)
Comment (jimmielin) [reply bld/namelist_files/use_cases/hist_geoschem_nudged.xml:35]: I think these are used for the CAM-chem's offline emissions module. We use HEMCO so these are just placeholders. Eventually when HEMCO is default all these XML options will be removed.

Comment (lizziel) [reply bld/namelist_files/use_cases/hist_geoschem_nudged.xml:35]: Got it, thanks!

File: bld/namelist_files/use_cases/hist_geoschem_nudged.xml (hunk 1 above)
Comment (jimmielin) [reply bld/namelist_files/use_cases/hist_geoschem_nudged.xml]: I kept this comment from `hist_geoschem.xml` but yes, this is the same in both `hist_trop_strat_nudged_cam6.xml` and `hist_trop_strat_vbs_cam6.xml`. I can update the comment if needed?

Comment (lizziel) [reply bld/namelist_files/use_cases/hist_geoschem_nudged.xml]: I prefer referencing the nudged file so it's clear what the analogous file is for all settings, including the nudged section.

Comment (jimmielin) [reply bld/namelist_files/use_cases/hist_geoschem_nudged.xml]: Thanks, references fixed. I noticed I actually added the referencing to `hist_trop_strat_nudged_cam6.xml` below where I added the nudged settings, but I updated the file header as well for clarity.

Comment (lizziel) [reply bld/namelist_files/use_cases/hist_geoschem_nudged.xml]: Thanks!

**Review by lizziel — APPROVED** [review]

**Pull request merged**